python/py-rawpy/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+name                py-rawpy
+version             0.17.2
+revision            0
+license             MIT
+
+github.setup        letmaik rawpy v0.17.2
+fetch.type          git
+
+python.versions     310
+
+maintainers         openmaintainer @jsalort
```

Comment:
please add an obfuscated e-mail address as well

Reply:
OK, I will have a look at how to do that.

Reply:
If you really don't want to list an email address, you can leave it as just a GitHub handle. It's just nice to be able to contact maintainers via email as well.